suites/reef/cephadm/tier1-remove-cluster.yaml

```diff
@@ -85,6 +85,12 @@ tests:
       module: test_client.py
       name: configure client
 
+  - test:
```

Comment:
@pranavprakash20 

Can we add same test to squid suite as well?

Rest look good

Reply:
Done